analyses/mdg/mdg_plague_exploration.md

```diff
@@ -852,10 +957,6 @@ Based on this simple analysis, using a threshold of at least 30 or 40 cases duri
 
 When using this method, the trigger would be reached in week 37 in 2017
 
-```python
```

Comment:
It is annoying I cannot comment on unchanged lines, but the plot on line 938-945 now is strange with the coloring and outdated title. Maybe do a quick update on that? 
Code starts with `alt.Chart(df_cap30_rolling_sum).mark_line().encode(`